test/unit_test/passes/qnn/test_qnn_context_bin_generator.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+from olive.model import SNPEModelHandler
+from olive.passes.olive_pass import create_pass_from_dict
+from olive.passes.qnn.context_binary_generator import QNNContextBinaryGenerator
+
+
+@patch("olive.passes.qnn.context_binary_generator.QNNSDKRunner")
+@pytest.mark.parametrize(
+    "config",
+    [
+        {
+            "backend": "libQnnHtp.so",
+        },
+        {
+            "backend": "libQnnHtp.so",
+            "extra_args": "--dlc_path extra_test.dlc",
+        },
+    ],
+)
+def test_snpe_model_input(mocked_qnn_sdk_runner, config, tmp_path):
```

Comment:
looks like this is not supported on Windows?

Reply:
Oh, true. I skip it in my local commit but forget to push. 🤣

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/7837e8f4-e695-47f7-8a4e-0c94ce403ac9)